src/commons/GitUtils.ts

```diff
@@ -15,6 +15,15 @@ export namespace GitUtils {
     const mergedTestsFiles: Set<string> = TestFilesUtils.getTestsFiles(getLastMergedFiles());
     return TestFilesUtils.extractTestIdFromFiles(mergedTestsFiles);
   }
+
+  /**
+   * Return list of all tests ids that merged by hours.
+   * @param hours - hours to count back
+   */
+  export function getMergedTestsIdsByLastHours(hours: number): Set<String> {
```

Comment:
this looks like a duplication of getLastMergedTestsIds function.
you can introduce an optional parameter to getLastMergedTestsIds and reuse the same function